modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/components/FieldsSidebarSettingsBody.js

```diff
@@ -44,6 +44,10 @@ const getColumn = ({objectFields}) => ({children, column, index}) => {
 					return <React.Fragment key={index} />;
 				}
 
+				if (objectFields.length && fieldName === 'indexType') {
+					return <React.Fragment key={index} />;
+				}
+
```

Comment:
@mateussgn,

Could you change the condition? Use something like this:

```
if (objectFields.length && (fieldName === 'repeatable' || fieldName === 'indexType')) {
     return <React.Fragment key={index} />;
}
```

Reply:
Also change the comment above the condition

`// Avoid using repeatable field when object storage type is selected`